requirements.txt

```diff
@@ -2,7 +2,6 @@ cached-property
 chainer>=4.0.0
 fastcache; python_version<'3.2'
```

Comment:
Since this is only necessary for Python below 3.2 and we say ChainerRL is tested with 3.5+, shouldn't we remove this?
I guess something similar applies to statistics.

Reply:
This import would need to be changed accordingly for fastcache: https://github.com/chainer/chainerrl/blob/e424a75c3c7560fb6d78594e70f52af9668788fb/chainerrl/functions/lower_triangular_matrix.py#L12

And here for funcsigs: https://github.com/chainer/chainerrl/blob/7523bb7a7f6ec4e4236875b6990837a9e7dcfd13/chainerrl/links/sequence.py#L13